docs/source/features/sheet_management/dataframe.rst

```diff
@@ -4,3 +4,139 @@
 ###########
  DataFrame
 ###########
+
+:class:`~graviti.dataframe.frame.DataFrame` is an integrated data structure
+with an easy-to-use API for simplifying data processing in Dataset.
+A Graviti DataFrame contains the 2-dimensional data with columns of Protex schema.
```

Comment:
`A Graviti DataFrame contains 2-dimensional tabular data and a Protex schema describing the names and types of each column.` 
Would this be more clear?